doc/templates/index.html

```diff
@@ -189,7 +189,8 @@ <h4 class="sk-landing-call-header">News</h4>
       <div class="col-md-4">
         <h4 class="sk-landing-call-header">Community</h4>
         <ul class="sk-landing-call-list list-unstyled">
-        <li><strong>About us:</strong> See <a href="about.html#people">authors</a> and <a href="{{ contributing_link }}" {{ contributing_attrs }}>contributing</a></li>
+        <li><strong>About us:</strong> See <a
+href="about.html#the-people-behind-scikit-learn">authors</a> and <a href="{{ contributing_link }}" {{ contributing_attrs }}>contributing</a></li>
```

Comment:
```suggestion
href="about.html#the-people-behind-scikit-learn">the people behind scikit-learn</a> and <a href="{{ contributing_link }}" {{ contributing_attrs }}>contributing</a></li>
```

Reply:
I worry that this is going to be too long and take too much space (this really matters, good front-page communication is short). If you want, I can put "people" instead of "authors"
![image](https://github.com/scikit-learn/scikit-learn/assets/208217/efdd6c75-be5c-4c49-82cf-738cbda92a32)

Reply:
Indeed. Let's use "people".

Reply:
Done